T7Suite/frmMain.cs

```diff
@@ -6585,6 +6585,7 @@ private double GetMapCorrectionFactor(string symbolname)
                 returnvalue = 1;
                 if (symbolname == "KnkSoundRedCal.fi_OffsMap") returnvalue = 0.1;
                 else if (symbolname == "IgnE85Cal.fi_AbsMap") returnvalue = 0.1;
+                else if (symbolname == "IgnNormCal.GasMap") returnvalue = 0.1;
```

Comment:
Response to comment above. Its just a correction factor that should be applied to all values. 
Same as resolution in the as2 files

*IgnNormCal.GasMap
MAP
1 10 0.000 450 -100 1 1 0 ø
IgnNormCal.m_AirXSP
MAF.m_AirInletIgn
IgnNormCal.n_EngYSP
In.n_Engine
Description:[" DESCRIPTION: Normal ignition map when Gas mode is active.
Gets activated when FlashSave.ST_GasHybridAct is set 
(gets set by Tech2 on gas hybrid cars) and one cylinder 
failed injection. In this case Lambda.ST_GasActive gets set.
At the same time diagnostic mode is forced into LOBD.
RANGE      : -10.0° to 45.0°
RESOLUTION : 0.1° "]